dashboard/app/models/sections/section.rb

```diff
@@ -129,27 +132,30 @@ def students_attributes=(params)
 
   # Adds the student to the section, restoring a previous enrollment to do so if possible.
   # @param student [User] The student to enroll in this section.
-  # @return [Follower] The newly created follower enrollment.
+  # @return [ADD_STUDENT_EXISTS | ADD_STUDENT_SUCCESS] Whether the student was already
+  #   in the section or has now been added.
```

Comment:
Is there a compelling reason to return strings rather than a boolean? It would seem more natural to return a boolean indicating whether the student was (newly) added to the section.